mmv1/third_party/terraform/services/dialogflow/resource_dialogflow_conversation_profile_test.go

```diff
@@ -0,0 +1,755 @@
+			},
+			{
+				ResourceName:            "google_dialogflow_conversation_profile.profile",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location"},
+			},
+			{
+				Config: testAccDialogflowConversationProfile_dialogflowAgentKnowledgebase2(context),
+			},
+			{
+				ResourceName:            "google_dialogflow_conversation_profile.profile",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location"},
+			},
+		},
+	})
+}
+func testAccDialogflowConversationProfile_dialogflowAgentKnowledgebase1(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+	resource "google_project" "agent_project" {
+		name = "tf-test-dialogflow-%{random_suffix}"
+		project_id = "tf-test-dialogflow-%{random_suffix}"
+		org_id     = "%{org_id}"
+		billing_account = "%{billing_account}"
+	}
+
+	resource "google_project_service" "agent_project" {
+		project = "${google_project.agent_project.id}"
+		service = "dialogflow.googleapis.com"
+		disable_dependent_services = false
+	}
+
+	resource "google_service_account" "dialogflow_service_account" {
+		account_id = "tf-test-dialogflow-%{random_suffix}"
+	}
+
+	resource "google_project_iam_member" "agent_create" {
+		project = "${google_project.agent_project.id}"
+		role    = "roles/dialogflow.admin"
+		member  = "serviceAccount:${google_service_account.dialogflow_service_account.email}"
+	}
+
+	resource "google_dialogflow_agent" "agent" {
+		project = "${google_project.agent_project.id}"
+		display_name = "tf-test-agent-%{random_suffix}"
+		default_language_code = "en-us"
+		time_zone = "America/New_York"
+		depends_on = [google_project_iam_member.agent_create]
+	}
+	
+	resource "google_dialogflow_conversation_profile" "profile" {
+		display_name  = "tf-test-conversation-profile-%{random_suffix}-new"
+		location = "global"
+		human_agent_assistant_config {
+			human_agent_suggestion_config {
+				feature_configs {
+					enable_event_based_suggestion = true
+					suggestion_feature {
+						type = "FAQ"
+					}
+					query_config {
+						context_filter_settings {
+							drop_handoff_messages       = false
+							drop_ivr_messages           = false
+							drop_virtual_agent_messages = false
+						}
+						knowledge_base_query_source {
+							knowledge_bases = ["projects/${google_project.agent_project.id}/locations/global/knowledgeBases/MTIxNTkzNjM1NzY3NjY2NjA2MDk"]
+						}
+						max_results = 1
+					}
+					suggestion_trigger_settings {
+						only_end_user = true
+					}
+				}
+			}
+			message_analysis_config {
+				enable_entity_extraction  = true
+				enable_sentiment_analysis = true
+			}
+		}
+		logging_config {
+			enable_stackdriver_logging = true
+		}
+		language_code = "en-US"
+	}
+	`, context)
+}
+func testAccDialogflowConversationProfile_dialogflowAgentKnowledgebase2(context map[string]interface{}) string {
```

Comment:
this is a very large config and can be hard to spot the difference between this and the first. Can you comment what's intended to change between the configs?